src/sinks/http.rs

```diff
@@ -138,35 +141,54 @@ impl SinkConfig for HttpSinkConfig {
     }
 
     fn input_type(&self) -> DataType {
-        DataType::Log
+        DataType::Any
     }
 
     fn sink_type(&self) -> &'static str {
         "http"
     }
 }
 
+#[derive(Serialize)]
+#[serde(untagged)]
+enum EventOrLog {
```

Comment:
I'm not seeing why this enum is required. Could we not just encode `Event`?

Reply:
This is yet another shim for backwards compatibility purposes. It will likely be removed since you've indicated backwards compatibility isn't something we're concerned with.